examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/ADM.java

```diff
@@ -16,22 +16,29 @@
     public static final SimpleDateFormat ADMDateFormat = new SimpleDateFormat("yyyy-MM-dd");
     public static final SimpleDateFormat ADMTimeFormat = new SimpleDateFormat("HH:mm:ss.SSS");
 
+    public static StringBuilder sbConstructor=new StringBuilder();
     public static String mkADMConstructor(String constructor, String content) {
-        return constructor + "(\"" + content + "\")";
+        sbConstructor.delete(0,sbConstructor.length());
```

Comment:
this made the `mkADMConstructor` non-functional. Why not just use the local variable?